cli/test/smokehouse/test-definitions/dobetterweb.js

```diff
@@ -272,22 +272,22 @@ const expectations = {
         details: {
           items: [
             {
-              url: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=100',
+              url: 'http://localhost:10200/dobetterweb/fcp-delayer.js?delay=5000',
             },
             {
-              url: 'http://localhost:10200/dobetterweb/unknown404.css?delay=200',
+              url: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=3000&capped',
             },
             {
               url: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=2200',
             },
             {
-              url: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=3000&capped',
+              url: 'http://localhost:10200/dobetterweb/dbw_tester.js',
             },
             {
-              url: 'http://localhost:10200/dobetterweb/dbw_tester.js',
+              url: 'http://localhost:10200/dobetterweb/unknown404.css?delay=200',
             },
             {
-              url: 'http://localhost:10200/dobetterweb/fcp-delayer.js?delay=5000',
+              url: 'http://localhost:10200/dobetterweb/dbw_tester.css?delay=100',
```

Comment:
Do you know why the order changes here? Looks like the order comes directly from TE so maybe some updated a while back applied a sort.